modules/installation-guide/partials/con_persistent-volume-configuration.adoc

```diff
@@ -5,95 +5,131 @@
 [id="storage-strategies-for-{prod-id-short}-workspaces_{context}"]
 = Storage strategies for {prod-id-short} workspaces
 
-Workspace Pods use Persistent Volume Claims (PVCs), which are bound to the physical Persistent Volumes (PVs) with link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes[ReadWriteOnce access mode]. It is possible to configure how the {prod-short} server uses PVCs for workspaces. The individual methods for this configuration are called PVC strategies:
+Storage strategy::
+A configurable method defining how {prod-short} workspaces uses persistent volume claims (PVCs) and persistent volumes (PVs). 
+It defines the storage for workspace data: projects, workspace logs, or additional Volumes defined by a user.
 
-include::example${project-context}-pvc-strategies.adoc[]
+.Storage strategies comparison
+[width="100%",cols="1,1,1,1",options="header"]
+|===
+| Storage strategy name
+| `common`
+| `per-workspace`
+| `unique`
 
-{prod-short} uses the `common` PVC strategy in combination with the "one {orch-namespace} per user" {orch-namespace} strategy when all {prod-short} workspaces operate in the user's {orch-namespace}, sharing one PVC.
+| Persistent volume count
+| One per user
+| One per workspace
+| Multiple persistent volumes per workspace
 
-[id="the-common-pvc-strategy_{context}"]
-== The `common` PVC strategy
+| Default
+| yes
+| no
+| no
 
-All workspaces inside a {platforms-namespace} use the same Persistent Volume Claim (PVC) as the default data storage when storing data such as the following in their declared volumes:
+| Control storage
+| easy
+| complex 
+| most complex
```

Comment:
I would probably remove this section because it is more ambiguous then infortative

Reply:
```suggestion
```